### DISCUSSION TITLE
Integration in Home Assistant

### DISCUSSION THREAD

**Discussion by pmolenaar** [serkri/SmartEVSE-3#96] [Q&A]

**Comment by dingo35** [comment]
Have you looked at https://github.com/dingo35/ha-SmartEVSEv3 

That might be much easier than configuring through configuration.yaml or node-red.

**Reply by pmolenaar** [reply]
I hadn't because I never saw it. Great - exactly what I needed!

**Comment by pmolenaar** [comment]
Dank je. Werkt super - wat een mooi werk! Eén ding: ik zie dat de SmartEVSE pingable moet zijn via mDNS. Omdat het bij mij op een apart IOT wifi draait (met access control op MAC adres basis), werkt dat niet. Is het cruciaal? Anders moet ik de SmartEVSE maar in het gewone netwerk hangen….

**Comment by dingo35** [comment]
If you want HA to communicate with your SmartEVSE, you better have them in the same LAN, or have the separate networks communicate with each other...

If mDNS doesn't work, you can enter an ip address as hostname also...
